training/trillium/Mixtral-8x7B-Pytorch/XPK/train.sh

```diff
@@ -37,4 +37,4 @@ python3 /workspace/transformers/examples/pytorch/language-modeling/run_clm.py \
 --block_size=${SEQUENCE_LENGTH} --optim=adafactor --save_strategy=no \
 --logging_strategy=no --fsdp="full_shard" \
 --fsdp_config=/app/fsdp_config.json --torch_dtype=bfloat16 \
---dataloader_drop_last=yes --max_steps=${MAX_STEP} --gmm --flash_attention
\ No newline at end of file
+--dataloader_drop_last=yes --max_steps=${MAX_STEP} --flash_attention --static
```

Comment:
Can you add a note on what `static` flag is doing? This seems to be not a standard arg in the HF upstream. Thanks

Reply:
Added a comment above.